packages/wonder-blocks-banner/src/components/__docs__/banner.stories.js

```diff
@@ -0,0 +1,355 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Banner from "@khanacademy/wonder-blocks-banner";
+import Button from "@khanacademy/wonder-blocks-button";
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import Link from "@khanacademy/wonder-blocks-link";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import BannerArgTypes from "./banner.argtypes.js";
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+
+export default {
+    title: "Banner",
+    component: Banner,
+    decorators: [
+        (Story: StoryComponentType): React.Element<typeof View> => (
+            <View style={styles.example}>
+                <Story />
+            </View>
+        ),
+    ],
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+    argTypes: BannerArgTypes,
+};
+
+const longText = `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
Yeah. @nishasy here are some examples that I think would be generic still, but create more distinction between each kind and also a clearer example:

- Info = "This is a message about something informative like an announcement."
- Success = "This is a message about something positive or successful!"
- Warning = "This is a message warning the user about a potential issue."
- Critical = "This is a message about something critical or an error."
- etc.

What do you think?

Reply:
Oh, I didn't know we weren't supposed to use lorem ipsum, but maybe that directive is just for shorter text, because, yeah, what would you use for longer text? I also like "Lorem Office-Speak-um" personally, but just because I think it's funny.

Reply:
@nishasy yeah, I think forcing it to be more narrow is also more realistic to what we would probably see. Especially on Mobile. 🙂 

The text examples I gave could be used for the "[Variants](https://5e1bf4b385e3fb0020b7073c-fdyhsmnhub.chromatic.com/?path=/story/banner--variants)" story/page you have, to further clarify their purpose/meaning.